pkg/spireentry/reconciler.go

```diff
@@ -110,11 +110,16 @@ func (r *entryReconciler) reconcile(ctx context.Context) {
 		}
 
 		clusterSPIFFEID.NextStatus.Stats.NamespacesSelected += len(namespaces)
+		var ignored = false
```

Comment:
This variable is not used outside of the scope of the for loop and should ideally be declared close to the point of use.